example-theme/extra-translations/en.edn

```diff
@@ -1,3 +1,3 @@
 {:t
  {:administration {:catalogue-items "Catalogue"}
-  :create-license {:license-text "Licence text"}}}
+  :create-license {:license-text "Text"}}}
```

Comment:
I thought the old one was quite sensible, as it was showcasing the difference between American and British spelling (license vs. licence). But I don't mind changing it.

Reply:
I found that it was a little bit confusing introducing another variant. And it's mainly just to test this feature.